src/Agent/NewRelic/Agent/Core/Config/Configuration.cs

```diff
@@ -4089,80 +4167,57 @@ public virtual configurationErrorCollectorIgnoreErrors Clone()
     [System.SerializableAttribute()]
     [System.ComponentModel.DesignerCategoryAttribute("code")]
     [System.Xml.Serialization.XmlTypeAttribute(AnonymousType=true, Namespace="urn:newrelic-config")]
-    public partial class configurationErrorCollectorIgnoreStatusCodes
+    public partial class ErrorMessagesTypeErrorClass
     {
         
-        private List<float> codeField;
+        private List<string> messageField;
+        
+        private string nameField;
         
         /// <summary>
-        /// configurationErrorCollectorIgnoreStatusCodes class constructor
+        /// ErrorMessagesTypeErrorClass class constructor
         /// </summary>
-        public configurationErrorCollectorIgnoreStatusCodes()
+        public ErrorMessagesTypeErrorClass()
```

Comment:
This name reads funny to me, and is harder for me to understand. I think that if we change the name of some of the types in the xsd, this generated name should become easier to understand? See the comments in the xsd file for some suggestions.